### PR TITLE
Added html for section 5

### PR DESCRIPTION
**Pull request by MatkovskyA**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -89,6 +89,37 @@ <h2 class="cashback__title">and cashback!</h2>
         <button class="cashback__button" id="order-open" type="button">Order a consultation</button>
       </div>
     </div>
+  </section>
+    <!-- Секция 4 -->
+  <section class="clients" id="clients">
+    <div class="clients__wrapper">
+      <ul class="clients__brands">
+        <li class="clients__brand">
+          <img src="./assets/brands/1.svg" alt="Brand 1">
```
Comment (Palych18) [inline index.html:98]: Stick to one style

**Review by Palych18 — APPROVED** [review]
Done

File: index.html [hunk 2]
```diff
@@ -89,6 +89,37 @@ <h2 class="cashback__title">and cashback!</h2>
         <button class="cashback__button" id="order-open" type="button">Order a consultation</button>
       </div>
     </div>
+  </section>
+    <!-- Секция 4 -->
```
[the remark this answers is not in the document]
Comment (MatkovskyA) [reply index.html:93]: Resolved

File: index.html (hunk 1 above)
Comment (MatkovskyA) [reply index.html:98]: Maded

**Pull request merged**